packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -131,12 +131,12 @@ $button-padding-ghost-sm: calc(0.375rem - 3px) 1rem !default;
 /// @type Number
 /// @access public
 /// @group button
-$button-border-width: 3px !default;
+$button-border-width: 2px !default;
 
 /// @type Number
 /// @access public
 /// @group button
-$button-outline-width: 1px !default;
+$button-outline-width: 2px !default;
```

Comment:
I think we just need to keep this at `1px`

Reply:
ok i got it

Reply:
I originally read it as 2px on both as well 😃

Reply:
We need to keep this at `1px`

Reply:
yeah i think once that is fixed we should be good^